app/src/main/java/com/jerboa/ui/components/settings/account/AccountSettings.kt

```diff
@@ -316,6 +322,24 @@ fun SettingsForm(
                     Text(text = stringResource(R.string.account_settings_show_scores))
                 },
             )
+            SwitchPreference(
```

Comment:
These settings are only added with 0.19.4? They should be behind a FeatureFlag?

Reply:
Fixed. The other ones should be okay, because you set it by default to null in your API addition.